ginga/misc/plugins/MultiDim.py

```diff
@@ -415,24 +455,23 @@ def set_naxis(self, idx, n):
             if image is None:
                 raise ValueError("Please load an image cube")
 
-            # invert index
-            naxislen = 2 + len(image.naxispath)
-            m = naxislen - (n+1)
+            m = n - 2
```

Comment:
@rirze, it would be best not to assume the loop variable has a certain value after exiting the loop (it would be different if it was a non-for variable).  I think it would be more readable and maintainable to say:
```python
n = len(dims) - 1
m = n - 2
```